src/components/EnergySeriesChart.tsx

```diff
@@ -67,7 +67,7 @@ export const EnergySeriesChart = ({ data }: OwnProps) => {
     };
 
     new Chart(context, { type: 'line', data: { labels, datasets }, options });
-  }, []);
+  }, [data]);
```

Comment:
Now the chart component listens to changes in the data and re-renders the chart with those changes